app/controllers/movies_controller.rb

```diff
@@ -38,7 +38,7 @@ def update
 
   def show
     @media = Movie.friendly.find(params[:id])
-    return redirect_to movie, status: :moved_permanently if request.path != movie_path(@media)
```

Comment:
This resolves [TMDB-MOVIEQUEUE-70](https://mike-vallano.sentry.io/issues/3904770136/?project=127874&query=is%3Aunresolved&referrer=issue-stream)